### ISSUE TITLE
Bug: Comb clears wires connected to power switch

### ISSUE THREAD

**Issue by jebob** [shanemadden/factorio-power-grid-comb#7]
Running the comb on a power switch connected to two electric poles clears the wires

Before

![image](https://user-images.githubusercontent.com/9222111/103161347-68b2fa00-47d8-11eb-87c6-982aa88a0ad8.png)

Before blueprint string

```
0eNqVkd1uwjAMhd/F1+nUBFghtzzGNE3QWsxS86P8rKuqvDtJ0BAMbrizo+PP5zgLHMeI1pEOIBeg3mgP8mMBTyd9GMtbmC2CBAqogIE+qNJZM6Fr/ESh/4bEgPSAvyB5+mSAOlAgvHBqM3/pqI7osuBKUDhQVA2O2AdHfWPNiJlvjc/DRpfNBcjfNgxmkJv3XKXEHojiuadH0vaPk+3mmDrvpZp2gX1sb83SUI1O5LDWbUm1j/y/ZnWveeZu9WJesb7PmxfXw8ubf2Lwg87XGbHl624nOtF2nHcipTNJNZhn
```

After

![image](https://user-images.githubusercontent.com/9222111/103161349-6f417180-47d8-11eb-8ac5-825f4474e518.png)

**Comment by jebob** [comment]
Options I see:
* Do not destroy & recreate any poles attached to a power switch
* When we recreate the poles, also recreate the power switch connection

**Comment by shanemadden** [comment]
Yeah, I think probably adding a special case to reconnect them in the same exact layout as they were before (in the same way the mod does now with circuit wires) will be the best way to fix it. I'll plan on digging around and see how tricky that will be to do.
